### PR TITLE
Fix TODO list generation for C#

### PR DESCRIPTION
**Pull request by MartinThoma**

### CODE REVIEW

File: 00_Utilities/TODO.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# TODO list
+ game | csharp | java | javascript | kotlin | lua | perl | python | ruby | rust | vbnet
+--- | --- | --- | --- | --- | --- | --- | --- | --- | --- | ---
+01_Acey_Ducey                  | ✅ | ✅ | ✅ | ⬜️ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅
+02_Amazing                     | ✅ | ✅ | ✅ | ⬜️ | ⬜️ | ✅ | ✅ | ✅ | ✅ | ✅
+03_Animal                      | ✅ | ✅ | ✅ | ✅ | ⬜️ | ✅ | ✅ | ✅ | rust | ✅
```
Comment (mojoaxel) [inline 00_Utilities/TODO.md:6]: Cool, but whats's up with the "rust" string in the table?

**Comment by mojoaxel** [comment]
@qequ What do you think?

**Review by mojoaxel — CHANGES_REQUESTED** [review]

File: 00_Utilities/TODO.md (hunk 1 above)
Comment (MartinThoma) [reply 00_Utilities/TODO.md]: It was a bug in the original script. I've just fixed it. It happened when the game-folder didn't contain the language folder.

**Review by mojoaxel — APPROVED** [review]
LGTM, but a minor markdown formating issue.

**Review by mojoaxel — CHANGES_REQUESTED** [review]
@MartinThoma I ran the script with `python3 ./markdown_todo.py` and it worked.
Strangely the result has some formatting issue at `20_Buzzword`. What am I doing wrong?

```markdown
16_Bug                         | ⬜️ | ✅ | ✅ | ⬜️ | ⬜️ | ⬜️ | ✅ | ✅ | ⬜️ | ✅
17_Bullfight                   | ✅ | ⬜️ | ✅ | ⬜️ | ⬜️ | ⬜️ | ✅ | ⬜️ | ⬜️ | ✅
18_Bullseye                    | ✅ | ✅ | ✅ | ⬜️ | ⬜️ | ✅ | ✅ | ⬜️ | ⬜️ | ✅
19_Bunny                       | ✅ | ✅ | ✅ | ⬜️ | ⬜️ | ✅ | ✅ | ✅ | ⬜️ | ✅
20_Buzzword | ✅ | ✅ | ✅ | ⬜️ | ⬜️ | ✅ | ✅ | ✅ | ⬜️ | ✅
21_Calendar                    | ✅ | ✅ | ✅ | ⬜️ | ⬜️ | ✅ | ✅ | ⬜️ | ⬜️ | ✅
22_Change                      | ✅ | ✅ | ✅ | ⬜️ | ⬜️ | ✅ | ✅ | ⬜️ | ⬜️ | ✅
23_Checkers                    | ✅ | ⬜️ | ✅ | ⬜️ | ⬜️ | ⬜️ | ✅ | ✅ | ⬜️ | ✅
24_Chemist                     | ✅ | ✅ | ✅ | ⬜️ | ⬜️ | ✅ | ✅ | ⬜️ | ⬜️ | ✅
```

**Pull request merged**